### PR TITLE
Login from facebook without using puppeteer

### PR DESCRIPTION
**Pull request by jzarca01**
I think this might work
Also we need to discuss how to implement the 2-step login with phone number

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -48,13 +47,20 @@ class TinderClient {
...
-      facebookEmailAddress: emailAddress,
-      facebookPassword: password,
-    }).then(accessToken => axios.create({
+    return axios({
+      method: 'GET',
+      url: 'https://b-graph.facebook.com/auth/login',
+      params: {
+        password: password,
+        email: emailAddress,
+        access_token: '350685531728|62f8ce9f74b12f84c123cc23437a4a32',
```
Comment (jaebradley) [inline src/index.js:56]: probably want to remove this?

File: src/index.js [hunk 2]
```diff
@@ -48,13 +47,20 @@ class TinderClient {
...
+        method: 'POST'
```
Comment (jaebradley) [inline src/index.js:57]: Should the `method` property on [line 51](https://github.com/jaebradley/tinder-client/pull/92/files#diff-1fdf421c05c1140f6d71444ea2b27638R51) be removed?

**Review by jaebradley — CHANGES_REQUESTED** [review]
@jzarca01 thanks for opening this PR!

Apologies for my lack of knowledge, but I'm unaware of the `b-graph.facebook.com/auth/login` endpoint - do you mind providing some context about what this change is supposed to do and how this endpoint works (perhaps even some documentation)?

**Comment by jzarca01** [comment]
I found the trick here https://github.com/Schmavery/facebook-chat-api/pull/684#issuecomment-425624643

Using this endpoint you can generate an access_token that can then be used to login into tinder
That way you won't need puppeteer anymore

File: src/index.js (hunk 2 above)
Comment (jaebradley) [reply src/index.js:57]: Ah, nvm, both this and the `access_token` value are explained by https://github.com/Schmavery/facebook-chat-api/pull/684#issuecomment-425624643

**Comment by jaebradley** [comment]
@jzarca01 that's a pretty neat workaround - let me verify!

**Comment by jaebradley** [comment]
@jzarca01 sorry about the late reply - it looks like [Tinder has changed it's auth workflow / API](https://github.com/fbessez/Tinder/issues/69). I've adjusted the existing implementation to work with this new API but it looks like the FB access token generated by workaround in this PR no longer is able to auth successfully with Tinder's new auth endpoint (the old auth endpoint is now just returning a `200` with the string `"OK"`).

I haven't done exhaustive testing or investigation so I could definitely be missing something. If you have any additional thoughts feel free to discuss them here, otherwise I'll probably end up closing this in the next week or so if I don't here from you.

Thanks again for your contributions and I'm sorry this change doesn't seem to work for the new Tinder auth flow.

**Pull request closed without merging**